api/db-cdr/generate-cloudsql-cdr.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This generates new cloudsql database for a cdr
+# note dev-up must be run to generate the schema
+# note run-local-data-migrations must be run to generate hard coded data from liquibase
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+# Make the vocabulary table from cdr with no changes
+#bq --project=all-of-us-ehr-dev cp test_merge_dec26.vocabulary test_vocabulary_ppi.vocabulary
+project="all-of-us-workbench-test"
+bqcdr="test_merge_dec26"
+cloudsql_instance=workbenchtest
+cdr_db_name=cdr
+gcs_bucket=gs://all-of-us-workbench-cloudsql-create
+
+# output files
+tmp_dir="/tmp"
+cdr_schema_file="cdr_schema.sql"
+public_schema_file="public_schema.sql"
+hard_data_file="hard_data.sql"
+
+# Dump cdr schema and copy to gcs
+echo "Dumping cdr schema "
+mysqldump -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} \
+    --no-data --add-drop-table --ignore-table=$cdr_db_name.DATABASECHANGELOG \
+    --ignore-table=$cdr_db_name.DATABASECHANGELOGLOCK $cdr_db_name --database cdr \
+    --add-drop-database > $tmp_dir/$cdr_schema_file
+gsutil cp $tmp_dir/$cdr_schema_file $gcs_bucket/$cdr_schema_file
+rm $tmp_dir/$cdr_schema_file
+
+# Dump the data from the hardcoded data tables -- criteria, db_domain... and copy to gcs
+echo "Dumping Hard coded data "
+mysqldump -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} \
+     --add-drop-table --disable-keys cdr db_domain criteria > $tmp_dir/$hard_data_file
+gsutil cp $tmp_dir/$hard_data_file $gcs_bucket/$hard_data_file
+rm $tmp_dir/$hard_data_file
+
+# Todo  maybe not hardcode service account name ?
+SERVICE_ACCOUNT=all-of-us-workbench-test@appspot.gserviceaccount.com
```

Comment:
Why do you need to activate the service account here, and use it below? Can't we just expect somebody to have run "gcloud auth login" before running this script? (That's how our other stuff works...) Or it looks like you set GOOGLE_APPLICATION_CREDENTIALS to the SA credentials in the docker-compose code... how does that tie into this?